.github/workflows/docker_release.yml

```diff
@@ -0,0 +1,116 @@
+name: Release to DockerHub
+on:
+  push:
+    tags:
+      - v*  # Launch DockerHub push on v* tags (e.g.: v1.2.1)
```

Comment:
I believe the v* needs to be in quotations, i.e. 'v*'
Without those quotations I was unable to get this action to trigger.

Reply:
Thanks for the quotes! :+1: